c/src/ml-api-inference-single.c

```diff
@@ -830,6 +830,36 @@ ml_single_open_custom (ml_single_h * single, ml_single_preset * info)
     goto error;
   }
 
+  if (nnfw == ML_NNFW_TYPE_NNTR_INF) {
```

Comment:
In my opinion, added code block should be moved to 805 line.

Reply:
Thank you! I found that this can't be moved to 805 line.

Calling `ml_single_set_input_info()` requires single_h->filter to be opened.

This makes the restriction that below procedure need to be called after `single_h->klass->start (single_h->filter)` (L827) Please correct me if I am missing something!


single_h->klass->start (single_h->filter)

Reply:
OK. LGTM!